src/utils/index.js

```diff
@@ -71,10 +71,26 @@ const validateUserDataValues = (totalCells, totalFOVs) => {
     return { userDataError, userDataErrorMsg };
 };
 
+const validateKeyInOptions = (options) => {
+    let optionKeyError = false;
+    let optionKeyErrorMsg = "";
+    const uniqueNames = new Set();
+    options.forEach((option) => {
+        if (uniqueNames.has(option.name) && !option.key) {
+            optionKeyErrorMsg = `Error: option name ${option.name} is not unique, key is required`;
+            optionKeyError = true;
```

Comment:
Does it make sense to end execution early here? If there are multiple non-unique options without keys, this will only return an error message for the last one in the array.

Reply:
Thanks for the fast feedback! My thought process was that by identifying one error at a time, we could ensure that data creators address the issue one by one. This might take the debugging process more straightforward. However, I totally see your point that highlighting all errors at once could provide a more comprehensive overview. 
I'm open to suggestions, what do you think would be the most effective method here?

Reply:
Just made changes to have all validation functions to return the first spotted error. Thank you for your insights!